src/mte_vatag.adoc

```diff
@@ -93,16 +93,18 @@ defined as below
 |Privilege level | LOWEST_VADDR_CURR_PRIV | HIGHEST_VADDR_CURR_PRIV
 |  U             | 0                      | 2 ^ (VADDR_BITS - 1) - 1
 |  S / HS / VS   | 2 ^ (VADDR_BITS - 1)   | 2 ^ (VADDR_BITS) - 1
-|  M             | 0                      | `Impl. defined`
+|  M             | `Impl. defined`        | `Impl. defined`
 |===
 
 VADDR_BITS is virtual addressing bits. VADDR_BITS can be 39, 48 or 57
 depending on Sv39, Sv48 or Sv57 virtual addressing mode in `satp` CSR.
+In bare mode, LOWEST_VADDR_CURR_PRIV and HIGHEST_VADDR_CURR_PRIV are
+implementation defined.
 
 `Svatag` depends on `Svukte` extension and thus assumes an even split in
 address space between user and supervisor with user address space spanning
 from zero to maxium positive address while supervisor address space spanning
 from minimum negative value to maximum negative value.
 
-Lowest virtual address in M-mode is defined to be 0 and highest virtual address
-for M-mode is implementation dependent.
+LOWEST_VADDR_CURR_PRIV and HIGHEST_VADDR_CURR_PRIV in M-mode are implementation
```

Comment:
Is the difference of "defined" and "dependent" meaningful? If not, maybe merge the two sentences?

Reply:
IF PA=VA, then the width of the PA == width of the VA which, for SV32 , is 32 bits (that one is weird, because it's really 34 bits of PA, but the upper 3/4 can't be accessed from Mmode without setting MPRV), 39/48/57 bits for SV39/48/57, etc. 
The only difference is that there is no sign extension , so  the max physical address = 2^(39/48/57)-1 isn't it?

Reply:
Memory tagging is dependent on pointer masking and both are applicable only for 64 bit. Thus SV32 is not applicable.

Sv39, Sv48 and Sv57 are essentially non-bare modes. Yes final generated physical addresses can be 2 bit width larger but from programmer's perspective they care for virtual addressing here. Even in the case of identity mapping when a `va` is identity mapped to `pa`, programmer doesn't know about those 2 extra bits.

Those 2 extra bits come into the picture only in case of accesses via M-mode or Bare mode and in those cases its implementation dependent.

Did I get your concern right?